docs/en_US/CommunitySharings/AutoCompletion.md

```diff
@@ -0,0 +1,39 @@
+# Auto Completion
```

Comment:
"Auto Completion" -> "Auto Completion of nnictl Commands"

Reply:
Updated.